client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -438,41 +460,42 @@ function mediaButton( editor ) {
 		editor.nodeChanged();
 	}
 
-	function toggleSizingControls( event ) {
-		if ( ! event.element || 'IMG' !== event.element.nodeName ) {
+	function toggleSizingControls( increase, event ) {
+		const selectedSite = getSelectedSiteFromState();
+		if ( ! event.element || 'IMG' !== event.element.nodeName || ! selectedSite ) {
 			return;
 		}
 
-		const parsed = MediaSerialization.deserialize( event.element );
+		const parsed = deserialize( event.element );
+		const media = assign( { width: Infinity, height: Infinity }, MediaStore.get( selectedSite.ID, parsed.media.ID ) );
 
-		// Disable sizing toggles if the image is transient
-		let isHidden = parsed.media.transient;
+		// Hide sizing toggles if the image is transient
+		const isHidden = !! parsed.media.transient;
+		this.classes.toggle( 'hidden', isHidden );
 
-		// Disable sizing toggles if the image is of an unknown size
-		if ( ! isHidden ) {
-			isHidden = ! parsed.appearance.size || ! includes( SIZE_ORDER, parsed.appearance.size );
+		// Disable decrease button when smaller than the smallest thumbnail size
+		// and the full size is bigger than the current size
+		if ( ! increase ) {
+			const thumb = MediaUtils.getThumbnailSizeDimensions( SIZE_ORDER[ 0 ], selectedSite );
+			const isDisabled =
+				( parsed.media.width || Infinity ) <= media.width &&
```

Comment:
What case would the parsed width be less than the thumbnail width but greater than the media's own width? More to the point, do we need this line?